assets/js/modules/reader-revenue-manager/datastore/publications.test.js

```diff
@@ -96,7 +94,7 @@ describe( 'modules/reader-revenue-manager publications', () => {
 					.dispatch( MODULES_READER_REVENUE_MANAGER )
 					.syncPublicationOnboardingState();
 
-				expect( syncStatus ).toBeUndefined();
+				expect( syncStatus ).toEqual( {} );
 			} );
 
 			it( 'should update the settings and call the saveSettings endpoint', async () => {
```

Comment:
This test should be renamed as the action is no longer saving the settings. I'd suggest something like this:

```js
			it( 'should call the `sync-publication-onboarding-state` endpoint and update the settings in state', async () => {
```

If renaming as suggested, please make the assertion for calling the endpoint more explicit:

https://github.com/google/site-kit-wp/blob/a6a2ffbf643d7509e14dead20270747c7e2100cc/assets/js/modules/reader-revenue-manager/datastore/publications.test.js#L126-L127